CDP4IMEInstaller/CDP4Dashboard.wxs

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi">
+
+  <?include variables.wxi ?>
+
+  <Fragment>
+    <ComponentGroup Id="CDP4DASHBOARD_CG" Directory="DASHBOARD">
+      
+      <Component Id="comp_CDP4Dashboard" Guid="{60DCB213-9D8E-4C5F-984F-94755EDB0397}" Location="local" Win64="$(var.Win64)">
+        <File Id="fil_CDP4Dashboard"  Vital="yes" Source="$(var.CDP4Dashboard.TargetPath)" KeyPath="yes"></File>
+      </Component>
+
+      <Component Id="comp_chartscore" Guid="{DDC8052B-365B-41D8-98CB-BCF0A2BC4EB8}" Location="local" Win64="$(var.Win64)">
+        <File Id="fil_chartscore" Vital="yes" Source="$(var.CDP4Dashboard.TargetDir)DevExpress.Charts.v19.2.Core.dll" KeyPath="yes" />
```

Comment:
any references to devexpress components need to be in the following file `CDP4IME.wxs` and not in the plugin specific parts.